validator/src/main/java/com/amazon/aoc/validators/CWMetricValidator.java

```diff
@@ -90,29 +92,47 @@ public void validate() throws Exception {
     RetryHelper.retry(
         maxRetryCount,
         () -> {
-          // We will query both the Service and RemoteService dimensions to ensure we get all
-          // metrics from all aggregations, specifically the [RemoteService] aggregation.
+          // We will query metrics that include Service, RemoteService, or RemoteTarget dimensions
+          // to ensure we get all metrics from all aggregations, specifically the [RemoteService] aggregation.
           List<String> serviceNames =
               Lists.newArrayList(
                   context.getServiceName(), context.getRemoteServiceDeploymentName());
-          // TODO - Put the following back in: "www.amazon.com",  "AWS.SDK.S3"
+          // TODO - Put the following back in: "www.amazon.com"
```

Comment:
I don't think we will put this back, so you can remove the comment entirely

Reply:
Removed